handlers/auth.go

```
@@ -61,105 +68,162 @@ func Login(c *fiber.Ctx) error {
+		return c.JSON(fiber.Map{
+			"code":    400,
+			"message": "Invalid JSON Sent",
+		})
+	}
+	if !comparePasswords(user.Password, []byte(data.Password)) {
+		return c.JSON(fiber.Map{
+			"code":    401,
+			"message": "Invalid Password",
+		})
 	}
 	db.Model(&user).Association("Sessions").Delete()
```

golangci-lint (1.62.2)

Error return value of `(*gorm.io/gorm.Association).Delete` is not checked

(errcheck)